OrbitGl/App.cpp

```diff
@@ -556,16 +556,15 @@ void OrbitApp::MainTick() {
   if (GTcpServer) GTcpServer->ProcessMainThreadCallbacks();
   if (GTcpClient) GTcpClient->ProcessMainThreadCallbacks();
 
+  // Tick Transaction manager only from client (OrbitApp is client only);
+  GOrbitApp->GetTransactionManager()->Tick();
```

Comment:
would it be better to have a separate thread for transaction manager?

Reply:
These are very low frequency events and the Tick method relays information to the TCPClient efficiently by using non-blocking calls that push data to a lockless queue that the TCP thread picks up.